<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportImageOptions</class>
 <widget class="QGroupBox" name="ExportImageOptions">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Resolution</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="imgWidthSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="value">
         <number>720</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="imgHeightSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="formatComboBox">
        <item>
         <property name="text">
          <string>PNG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JPG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BMP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTransparency">
        <property name="text">
         <string>Transparency</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
